apps/demo-pkp-social-auth-next-ts/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
let's try to unify to use `yarn.lock` throughout the project, no need for both `package-lock.json` and `yarn.lock`.